featuretools/primitives/install.py

```diff
@@ -100,12 +98,20 @@ def download_archive(uri):
     local_archive = os.path.join(get_installation_temp_dir(), filename)
 
     with open(local_archive, 'wb') as f:
-        try:
+        if uri.startswith('s3://') or uri.startswith('https://s3'):
+            try:
+                import s3fs
+                from botocore.exceptions import NoCredentialsError
+            except ImportError:
+                raise ImportError("The s3fs library is required to handle s3 files")
```

Comment:
s3fs is actually only used as a fallback since `smart_open` cannot handle the anonymous clients which leads to the `NoCredentialsError` case. smart open can open s3 files in other cases.

see this issue: https://github.com/RaRe-Technologies/smart_open/issues/250

perhaps move this try/except into the `NoCredentialsError` block which is the only place s3fs is used.

Reply:
I see. I was assuming that `smart_open` also uses `s3fs` under the hood. I've now placed the `s3fs` import as you suggested.